doc/rst/drivers.rst

```diff
@@ -35,7 +38,6 @@ feature-set/benefits:
     - Supports Windows sound APIs such as WASAPI, DirectSound, MME
     - Supports high-end audio features such as bit-perfect PCM up to 768 kHz sampling rate, native DSD format (through ASIO) up to DSD1024
     - Supports multiple clock sources such as S/PDIF, ADAT or WCLK inputs
-    - Supports MIDI 1.0 class, including MIDI port sharing
```

Comment:
They still support his, we just don't use it for testing

Reply:
fixed it.